src/pages/home/Home.scss

```diff
@@ -0,0 +1,138 @@
+@import '~theme/base/variables';
+@import '~theme/base/utils';
+
+$distance-height-variation-before: 20px;
+$distance-height-variation-after: 40px;
+
+// Sections with the text at the right
+@mixin isReverseSection() {
+    text-align: right;
+
+    header {
+        &:after {
+            right: 0;
+        }
+    }
+    
+    @media screen and (min-width: 768px) {
+        > div {
+            flex-direction: row-reverse;    
+        }
+    }
+}
+
+.hero {
+    position: relative;
+    min-height: 100vh;
+    padding-top: 6rem;
+    padding-bottom: 3rem;
+
+    @media screen and (min-width: 768px) {
+        min-height: 550px;
+    }
+
+    &:before {
+        content: '';
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        background: url('./assets/images/main-header-background.png') no-repeat;
```

Comment:
I think we should use SVG file here, just like we are doing it for all other images like `about.svg`, `mission.svg`, `meeting.svg`, etc.

Reply:
There's a PR already for this, just leave this as it is.